code/campaign/build.gradle

```diff
@@ -66,6 +66,12 @@ android {
         targetCompatibility rootProject.ext.targetCompatibility
     }
 
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8
```

Comment:
nit: You can define these as variables in ../build.gradle and reference them here
https://github.com/adobe/aepsdk-core-android/blob/main/code/build.gradle#L46